### ISSUE TITLE
Positions Incorrect With Window Scrolling 

### ISSUE THREAD

**Issue by PeterMRichards** [AhmedAyachi/LinkerLine#5]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `linkerline@1.3.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/linkerline/src/index.js b/node_modules/linkerline/src/index.js
index 40e9a18..2504222 100644
--- a/node_modules/linkerline/src/index.js
+++ b/node_modules/linkerline/src/index.js
@@ -33,7 +33,10 @@ export default class LinkerLine extends LeaderLine {
         super.position();
         const {element}=this,parent=element.parentNode;
         const {left,top}=parent.getBoundingClientRect();
-        element.style.transform=`translate(${parent.scrollLeft-left}px,${parent.scrollTop-top}px)`;
+
+        const scrollX = parent.scrollTop-left - window.scrollX;
+        const scrollY = parent.scrollTop-top - window.scrollY;
+        element.style.transform=`translate(${scrollX}px,${scrollY}px)`;
     }
 
     show(effectName,options){
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by AhmedAyachi** [comment]
Hello @PeterMRichards, thanks.
Can you provide a screenshot.
and I guess: 
```
//this
const scrollX = parent.scrollTop-left - window.scrollX;
//should be
const scrollX = parent.scrollLeft-left - window.scrollX;
```

**Comment by kdagnan** [comment]
I was also having this issue, though I could not for the life of me figure out what was causing it because it seemed so inconsistent. The changes provided (with @AhmedAyachi 's update) are working as intended now.

**Comment by AhmedAyachi** [comment]
@kdagnan @PeterMRichards 
It's going to be fixed in version 1.4.0, it's already implemented in the dev branch.
